elf-regressions/jalr/test.s

```diff
@@ -0,0 +1,18 @@
+.section .text.init
```

Comment:
This passes and is the same program in jalr_custom -- its here to show that changing the entrypoint, but keeping the same ROM causes rom-setup to fail